app/models/admin/cohort.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::Cohort < ActiveRecord::Base
+  self.table_name = 'admin_cohorts'
+  belongs_to :project, :class_name => 'Admin::Project'
+  has_many :items, :class_name => 'Admin::CohortItem', :foreign_key => "admin_cohort_id"
```

Comment:
Can you add a dependent destroy (or whatever it is called) here. This way if a cohort is deleted, then all of the related cohort items will be removed too.